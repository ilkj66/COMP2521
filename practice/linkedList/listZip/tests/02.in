7 2 8
5 1 4 9 3 6
