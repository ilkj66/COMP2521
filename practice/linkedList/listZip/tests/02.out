Enter list 1: Enter list 2: 
List 1: [7] -> [2] -> [8] -> X
List 2: [5] -> [1] -> [4] -> [9] -> [3] -> [6] -> X
Zipped: [7] -> [5] -> [2] -> [1] -> [8] -> [4] -> [9] -> [3] -> [6] -> X
